frontend/BossAlienMediaPlayer/src/app/http.service.ts

```diff
@@ -26,13 +26,11 @@ export class HttpService {
     this.lastRequestTime = Date.now();
     this.retryDelay = retryDelay;
     let http$ = this.http.get<T>(url, {observe: "response"});
-    http$.pipe(
```

Comment:
@kevadsett I've removed this bit and moved it after the results arrive. I checked with print debugging and this pipe was never executed, so the version was never checked. Once moved inside the subscribe it worked as intended.